examples/dtls-echo/dtls-client.c

```diff
@@ -126,20 +128,15 @@ static void dtls_handle_read(dtls_context_t *ctx)
         session.ifindex = remote.netif;
     }
 
-    if (memcpy(&session.addr, &remote.addr.ipv6, 16) == NULL) {
-        puts("ERROR: memcpy failed!");
-        return;
-    }
+    memcpy(&session.addr, &remote.addr.ipv6, 16) == NULL);
```

Comment:
typo :wink:

Reply:
And magic number!

Reply:
yeah missed that. Fixed in latest commit.